packages/redux/src/analytics/middlewares/wishlist.ts

```diff
@@ -12,23 +12,34 @@ import {
   actionTypes as wishlistActionTypes,
 } from '../../wishlists';
 import { logger } from '@farfetch/blackout-analytics/utils';
+import {
+  WishlistActionMiddlewareOptions,
+  WishlistActionProcessedOptions,
+  WishlistProductUpdateSetActionMetadata,
+} from './types';
 import Analytics, {
   eventTypes,
   fromParameterTypes,
 } from '@farfetch/blackout-analytics';
 import get from 'lodash/get';
 import isNil from 'lodash/isNil';
+import type { AnyAction, Dispatch, Middleware } from 'redux';
+import type { StoreState } from '@farfetch/blackout-redux/types';
```

Comment:
Use relative-imports as this module is already in @farfetch/blackout-redux package.